.drone/drone.jsonnet

```diff
@@ -707,6 +707,8 @@ local manifest_ecr(apps, archs) = pipeline('manifest-ecr') {
         image: 'alpine',
         depends_on: ['check-version-is-latest'],
         commands: [
+          'apk add --no-cache bash git',
```

Comment:
Out of interest, can we not just use `ash` instead?

Reply:
You mean `sh` instead of `bash`?

```
+ RELEASE_TAG=$(./tools/image-tag)
env: can't execute 'bash': No such file or directory
```

I guess our `image-tag` requires `bash` looks like.

Reply:
No, I mean `ash` which is the default shell of Alpine

Reply:
We have `#!/usr/bin/env bash` in the header of `image-tag`, which is probably unnecessary.

Reply:
> We have #!/usr/bin/env bash in the header of image-tag, which is probably unnecessary.

yep, nothing bash specific is used in the script )) so I believe we can remove the header from the script or make it flexible to be able to run in any terminal...

anyway, not only bash was missed, `git` is also required, so let's merge this PR and update `./tools/image-tag` script when we have time and need  ))

Reply:
SGTM 👍